util/project.mplstyle

```diff
@@ -0,0 +1,39 @@
+# Increase default linewidth
+lines.linewidth: 2.0      # line width in points
+
+# Increase default legend line-length
+legend.handlelength:  3.5 # the length of the legend lines
+
+# Default marker size
+lines.markersize:      6  # marker size, in points
+
+# set legend location
+legend.loc:           best
+
+# Set default font size, everything scales with this
+font.size:    20.0
+
+# Relative font sizing for various things (axes, ticks, legend and titles)
+axes.titlesize:     medium  # font size of the axes title
+axes.labelsize:     medium  # font size of the x and y labels
+xtick.labelsize:    x-small # font size of the tick labels
+ytick.labelsize:    x-small # font size of the tick labels
+legend.fontsize:    10
+figure.titlesize:   large   # size of the figure title (``Figure.suptitle()``)
+figure.labelsize:   large   # size of the figure label (``Figure.sup[x|y]label()``)
+
+# Use latex for rendering, set font to computer modern
+text.usetex:        True # use latex for all text handling
+mathtext.fontset:   cm   # change from 'dejavusans' (default)
+font.family:        cm   # Make the default font consistent
+
+# Set a default group for line colors and dash styles
+axes.prop_cycle: cycler('color', [ "#EE2E2F", "#185AA9", "#008C48", "#F47D23", "#662C91", "#A21D21", "#B43894", "#010202"]) + cycler('linestyle',  [ [], [10, 5], [10, 4, 3, 4], [3, 3], [10, 4, 3, 4, 3, 4], [3, 2,6,2], [3,4,1,4], [1,1] ])
```

Comment:
I got a complaint from matplotlib that

> Bad value in file 'project.mplstyle', line 31 ('axes.prop_cycle: cycler(\'color\', [ "#EE2E2F", "185AA9", "008C48", "F47D23", "662C91", "A21D21", "B43894", "010202"]) + cycler(\'linestyle\',  [ [], [10, 5], [10, 4, 3, 4], [3, 3], [10, 4, 3, 4, 3, 4], [3, 2,6,2], [3,4,1,4], [1,1] ])'): Key axes.prop_cycle: 'cycler('color', [ "' is not a valid cycler construction: EOL while scanning string literal (<unknown>, line 1)

I think this got fixed in later versions, but it is interpreting the `#` in `#EE2E2F` as a comment, so it's not reading the rest of the line.  However, by removing the `#` marks, my version of matplotlib still knows to read those as colors.